org/all-prs.ts

```diff
@@ -1,9 +1,25 @@
-import { schedule } from "danger"
+import { schedule, danger, warn, fail } from "danger"
 
-// Check for dependency changes
+const pr = danger.github.pr
+const bodyAndTitle = (pr.body + pr.title).toLowerCase()
+
+// Custom modifiers for people submitting PRs to be able to say "skip this"
+const wipPR = bodyAndTitle.includes("wip ") || bodyAndTitle.includes("[wip]")
```

Comment:
My thought here is that I'd rather see this check only the title. If I say something about WIP in the body, it could be unrelated, but if I say WIP in the title, then yeah skip stuff.

Reply:
What are the thoughts about parsing natural text vs using e.g. labels?

Reply:
I'm hesitant on labels as it's hard to keep all labels in sync (https://github.com/danger/peril/issues/138) and it feels like an unfamiliar action - the WIP in title takes existing behavior people do and codifies it. While saying "Use a label" is an entirely new flow for people.

Reply:
Makes sense 👍